src/lib/cucumberjs/cucumber.js

```diff
@@ -95,7 +95,7 @@ class Cucumber {
         }
 
         const failWhenNoTestsRun = booleanHelper.isTruthy(this.options['fail-when-no-tests-run']);
-        const noTestsFound = JSON.parse(jsonResults).length === 0;
+        const noTestsFound = jsonResults === null || JSON.parse(jsonResults).length === 0;
```

Comment:
Not sure what the implications are if `jsonResults === null` (when passing it as an argument to callback on L100). Currently, this throws because `JSON.parse(null).length`